src/meetings/components/sidebar/visualEffectsAccordion/VisualEffectsList.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { FC, useMemo } from 'react';
+
+import { Container, Icon, Text } from '@zextras/carbonio-design-system';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import useVirtualBackground from '../../../../hooks/useVirtualBackground';
+import { getBackgroundImage } from '../../../../store/selectors/ActiveMeetingSelectors';
+import useStore from '../../../../store/Store';
+import { VirtualBackgroundType } from '../../../../types/store/ActiveMeetingTypes';
+import { getBackgroundChunks } from '../../../../utils/MeetingsUtils';
+
+type VisualEffectCardsProps = {
+	meetingId: string;
+};
+
+const PictureContainer = styled(Container)<{ $picture?: string | false; $isSelected: boolean }>`
+	background-size: cover;
+	background-position: center;
+	aspect-ratio: 1.2959;
+	border-radius: 0.5rem;
+	${({ $isSelected, theme }): string | false =>
+		$isSelected && `border: 1px solid ${theme.palette.success.regular};`}
+	${({ $picture, theme }): string =>
+		$picture
+			? `background-image: url(${$picture});`
+			: `background-color: ${theme.palette.gray0.regular}`};
+`;
+
+const ListContainer = styled(Container)`
+	overflow-y: scroll;
+`;
+
+const VisualEffectsList: FC<VisualEffectCardsProps> = ({ meetingId }) => {
+	const [t] = useTranslation();
```

Comment:
Here it's better to check because it could easily be resolved with CSS grid layout with something like this:

  display: grid;
  grid-template-columns: repeat(3, 1fr);

and passing as first 2 elements the none and blur and following all the backgrounds available